cmds/exp/pox/pox.go

```diff
@@ -94,31 +104,33 @@ import (
 
 const usage = "pox [-[-debug]|d] -[-run|r] | -[-create]|c  [-[-file]|f tcz-file] file [...file]"
 
+type mp struct {
+	source string
+	target string
+	fstype string
+	flags  uintptr
+	data   string
+	perm   os.FileMode // for target in the chroot
+}
+
 var (
 	debug  = flag.BoolP("debug", "d", false, "enable debug prints")
 	run    = flag.BoolP("run", "r", false, "Run the first file argument")
 	create = flag.BoolP("create", "c", false, "create it")
 	zip    = flag.BoolP("zip", "z", false, "use zip instead of squashfs")
 	file   = flag.StringP("output", "f", "/tmp/pox.tcz", "Output file")
+	extra  = flag.StringP("extra", "e", "", "extra binds to do, comma-separated, in the form source:target[,source:target]")
```

Comment:
Can we add an explanation/example with -e to the file comment and the usage message?

Reply:
see what you think, I added more.